2-other-html-tags/index2.html

```diff
@@ -0,0 +1,28 @@
+    <p>En Muppler encontrarás el mejor artista para tu matrimonio</p>
+    <h3>Cumpleaños</h3>
+    <h3>Conciertos</h3>
+    <h2>Artistas</h2>
+    <h3>Binomio de oro</h3>
+    <h3>Silvestre dangond</h3>
+    <ul>
+        <li>HOME</li>
+        <LI>ARTISTAS</LI>
+        <LI>EVENTOS</LI>
+        <LI>CONTACTO</LI> 
```

Comment:
```suggestion
        <li>ARTISTAS</li>
        <li>EVENTOS</li>
        <li>CONTACTO</li> 
```

Reply:
Remember that you can link your headers with the links using the `id` attribute.